### DISCUSSION TITLE
DB Navigator: filter on Tables affects Views tree.

### DISCUSSION THREAD

**Discussion by tatzio** [dbeaver/dbeaver#17530] [Q&A]
`v22.1.5 / Win10 / DB2 z/OS`

I'm not sure if this is intended or is it a bug.

When I apply filter on Tables tree - tables are filtered correctly, BUT Views subtree suddenly stops displaying anything.
When I remove Tables filter all is back to normal.
![image](https://user-images.githubusercontent.com/106803609/187923667-eb2acee8-ee23-464d-b12a-661918af1d88.png)

**Comment by LonwoLonwo** [comment]
Hello @tatzio 

What kind of filter did you use?

**Reply by tatzio** [reply]
> Hello @tatzio
> 
> What kind of filter did you use?

Does not matter really. Even if I try the simplest one like `[text]*` - tables are filtered OK but Views tree is empty.
![image](https://user-images.githubusercontent.com/106803609/188225078-a317a268-34cd-456f-89e8-3d01f293042c.png)
